django/utils/encoding.py

```diff
@@ -152,12 +152,12 @@ def iri_to_uri(iri):
     portion that is suitable for inclusion in a URL.
 
     This is the algorithm from section 3.1 of RFC 3987.  However, since we are
-    assuming input is either UTF-8 or unicode already, we can simplify things a
-    little from the full method.
+    assuming input is a string already, we can simplify things a little from
```

Comment:
Small reword: "This is the algorithm from section 3.1 of RFC 3987, slightly simplified since the input is assumed to be a string rather than ???." (same wording could be used in docs)